newIDE/app/src/Utils/GDevelopServices/Release.js

```diff
@@ -8,14 +8,18 @@ export type Release = {
   description: ?string,
 };
 
-export const getReleases = (): Promise<Array<Release>> => {
-  return axios
-    .get(`${GDevelopReleaseApi.baseUrl}/release`, {
-      params: {
-        last: 4,
-      },
-    })
-    .then(response => response.data);
+export const getReleases = async (): Promise<Array<Release>> => {
+  const response = await axios.get(`${GDevelopReleaseApi.baseUrl}/release`, {
+    params: {
+      last: 4,
+    },
+  });
+  const releases = response.data;
+  if (!Array.isArray(releases)) {
```

Comment:
for some reason, the response is sometimes not an array and calling `.find()` on it causes a crash